src/lib.rs

```diff
@@ -125,6 +125,25 @@
 //! let encoded = to_vec(&programming_languages);
 //! assert_eq!(encoded.unwrap().len(), 103);
 //! ```
+//!
+//! Serializing a `Vec` as a specialized byte string uses about 2x less RAM and
+//! 100x less CPU than as a default array.
```

Comment:
Can you be more specific here? I think you mean "less CPU time" but I was a little confused when reading.